### DISCUSSION TITLE
Using Tabpy on Tableau Reader

### DISCUSSION THREAD

**Discussion by dannikaaa** [tableau/TabPy#565] [Q&A]
Hi I was wondering is it possible to use tabpy on tableau reader? Thanks!!

**Comment by nmannheimer** [comment]
Unfortunately it is not at this time. What are you looking to do with plugging TabPy into reader?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Danica ***@***.***>
Sent: Tuesday, August 30, 2022 9:22:22 PM
To: tableau/TabPy ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [tableau/TabPy] Using Tabpy on Tableau Reader (Discussion #565)


Hi I was wondering is it possible to use tabpy on tableau reader? Thanks!!

**Reply by dannikaaa** [reply]
ah ok thank you! I was looking into alternatives for tableau server as I don't have the resources to build a separate server for tabpy. 

Could you suggest any alternative ways? thanks!

**Comment by nmannheimer** [comment]
Can you connect TabPy to an existing tableau server? TabPy can connect to desktop, server, prep, and Online. TabPy also requires very little resources of its own

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by dannikaaa** [reply]
I tried installing it to the tableau server and it crashed the whole thing, also I asked on tableau community and they said I had to create a new server for it

**Comment by nmannheimer** [comment]
Ah yes, you don’t want to install TabPy on the same machine where tableau server is running. But TabPy is very light weight so you can install it on almost any other machine and connect it to server or desktop.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by dannikaaa** [reply]
hmm ok thank you!
